styles/card.css

```diff
@@ -0,0 +1,153 @@
+.card {
+  background: var(--cardBackground);
+  width: 20rem;
+  border-radius: 0.5rem;
+  cursor: pointer;
+  border: 1px solid transparent;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.card-box-shadow {
+  box-shadow: 0 4px 4px 0 rgba(1, 1, 0.5, 0.05);
+}
+
+.horizontal-card-img {
+  width: 10rem;
+  height: 8rem;
+  border-top-left-radius: 0.5rem;
+}
+
+.card-img {
+  width: 20rem;
+  height: 12rem;
+  border-top-left-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
+}
+
+.card-section {
+  display: flex;
+  flex-direction: column;
+}
+
+.card-section .card-header {
+  padding: 0.3rem 0.8rem;
+  overflow-wrap: break-word;
+}
+
+.horizontal-card-section {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
+
+.card-desc {
+  padding: 1rem;
+  overflow-wrap: break-word;
+  font-size: 1rem;
+  order: 3;
+  color: var(--cardTextColor);
+}
+
+.card-section:hover {
+  background: var(--cardSectionHoverColor);
+}
+
+.card-header-title {
+  font-size: 1.25rem;
+}
+
+.author-name {
+  margin-bottom: 1rem;
+  margin-top: 0.2rem;
+}
+
+.card-title {
+  padding-left: 1rem;
+  padding-top: 1rem;
+}
+
```

Comment:
you can also create utility classes for padding and margins.

Reply:
I am not using utility classes for padding, margins, and flex because it's affecting my components now. Will use it in another project. Although I used it for font-weight etc.

Reply:
Okay